package/MDAnalysis/topology/PDBParser.py

```diff
@@ -241,6 +244,8 @@ def _parseatoms(self):
                 altlocs.append(line[16:17].strip())
                 resnames.append(line[17:21].strip())
                 chainids.append(line[21:22].strip())
+                #Saving the elements type in a list
+                elements.append(line[76:78].strip())
```

Comment:
This column isn't guaranteed in PDB files -- MDAnalysis itself doesn't write it out (fixed in #2609 because it was breaking DSSP). You need to check the line length to avoid IndexErrors.

Reply:
Python lets you slice a string beyond it's end ie `'cat'[100:110] == ''`, so these will just be blank strings if the column isn't there

Reply:
Yes, I missed that. So, an alternative can be:
`if not any(elements):
     do (element parsing)
else if not any(atomnames):
     # Guess elements types from Atom names
     do (same task of parsing to get elements)
else:
     return False for element type`

Does this sound reasonable?